### PR TITLE
fix: Social media tests

### PR DESCRIPTION
**Pull request by alainajaffer**

### CODE REVIEW

File: site/pages/api/hootsuite-callback.api.ts [hunk 1]
```diff
@@ -44,7 +44,7 @@ const hootsuiteCallback = async (req: NextApiRequest, res: NextApiResponse) => {
         if (!idToken?.Parameter?.Value) {
             throw new Error("idToken must be defined");
         }
-        const decodedToken = jose.decodeJwt(idToken?.Parameter?.Value);
```
Comment (alvak1) [inline site/pages/api/hootsuite-callback.api.ts:47]: Was this change intentional

Comment (alainajaffer) [reply site/pages/api/hootsuite-callback.api.ts:47]: yes no need to import all of it

**Review by alvak1 — APPROVED** [review]
Approve
